src/drivers/intel/ssp/ssp.c

```diff
@@ -686,6 +700,46 @@ static int ssp_pre_start(struct dai *dai)
 
 	dai_info(dai, "ssp_set_config(), sscr0 = 0x%08x", sscr0);
 out:
+	if (!ret)
```

Comment:
nit-pick: for consistency with the rest of the code, use if (ret < 0) ?

Reply:
The if (ret < 0) in `ssp_mclk_prepare_enable` is to show some error message, but we already show error message in each error case. We want to set SSP_CLK_BCLK_ACTIVE bit if `ret == 0`